internal/metamorphic/crossversion/crossversion_test: TestMetaCrossVersion failed

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7593487?buildTab=log) with [artifacts](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7593487?buildTab=artifacts#/) on refs/heads/master @ [b9289d765e2c](https://github.com/cockroachdb/pebble/commits/b9289d765e2c):


```
        db.Get("rnnxkc@8")
        db.Set("vvgeqgyshc@17", "oxywfjsbppwuemc")
        iter60.SeekGE("eoqa@12", "")
        iter51.Prev("")
        iter55.SeekPrefixGE("tybvldvdnf@8")
        iter52.SeekPrefixGE("duoagoyrgm@14")
        db.Get("sewumgnvhlo@8")
        db.Delete("sxars@9")
        db.Set("slcvaohadk@13", "oozfhgoibmaahoted")
        db.DeleteRange("beiz@1", "caajvcqwjb@15")
        iter60.First()
        iter52.SetBounds("yrmxxcu@9", "yrmxxcu@9")
        iter52.SeekLT("yrmxxcu@9", "")
        iter52.Next("")
        iter52.Prev("")
        iter52.First()
        iter52.First()
        db.Delete("zhllqdad@2")
        db.Set("jzayqrmidx@8", "vljew")
        iter51.Prev("")
        db.Get("vpeedpqdde@8")
        iter60.SeekLT("ottj@13", "")
        iter60.SeekGE("lofffbpsq@11", "nzoab@5")
        db.Delete("mmlygsxxfk@14")
        iter56.First()
        iter51.SeekLT("xeqmml@3", "nlqcg@6")
        db.DeleteRange("ceoujw@8", "onslgcymo@12")
        iter60.Last()
        iter53.First()
        iter58.Last()
        iter58.First()
        iter60.SeekLT("proo@2", "mpcdpzvxydx@7")
        db.Merge("covr@13", "negkwkzhywhzuqu")
        iter60.SeekGE("zrytgxuhw@8", "")
        db.Delete("meophwrc@17")
        db.Get("ceoltdy@8")
        db.SingleDelete("tedv@13", true /* maybeReplaceDelete */)
        iter60.SeekGE("lwkmqbcztqc@11", "")
        iter58.First()
        iter51.Next("")
        iter53.Last()
        iter52.SeekPrefixGE("ztsiclpmz@12")
     === RUN   TestMeta/compare
    history.go:181: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/history.go:181
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/meta_test.go:463
        	Error:      	Received unexpected error:
        	            	open /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion3717417490/001/1668835203607951046/c4d8b2a0_1473996349973821806_009/_meta/221119-052024.6841400114522/standard-000/history: no such file or directory
        	Test:       	TestMeta/compare
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=210987648325619438-opts=standard-011(sha=4ddacdaf-seed=1397275708202510285-opts=random-018(<empty>)) (46.72s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/test-eng
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7636744?buildTab=log) with [artifacts](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7636744?buildTab=artifacts#/) on refs/heads/master @ [6ee5cca60f68](https://github.com/cockroachdb/pebble/commits/6ee5cca60f68):


```
                --- PASS: TestMeta/execution/random-018 (1.70s)
                --- PASS: TestMeta/execution/random-015 (1.12s)
                --- PASS: TestMeta/execution/random-022 (2.86s)
                --- PASS: TestMeta/execution/random-019 (2.99s)
                --- PASS: TestMeta/execution/random-013 (1.09s)
                --- PASS: TestMeta/execution/random-011 (1.04s)
                --- PASS: TestMeta/execution/random-010 (1.22s)
                --- PASS: TestMeta/execution/random-016 (2.91s)
                --- PASS: TestMeta/execution/random-006 (0.91s)
                --- PASS: TestMeta/execution/random-009 (1.68s)
                --- PASS: TestMeta/execution/random-017 (4.56s)
                --- PASS: TestMeta/execution/random-007 (1.16s)
                --- PASS: TestMeta/execution/random-005 (0.35s)
                --- PASS: TestMeta/execution/random-004 (0.40s)
                --- PASS: TestMeta/execution/random-003 (0.47s)
                --- FAIL: TestMeta/execution/random-001 (0.26s)
                --- PASS: TestMeta/execution/random-023 (5.94s)
                --- PASS: TestMeta/execution/random-002 (0.57s)
                --- PASS: TestMeta/execution/random-014 (4.06s)
                --- PASS: TestMeta/execution/random-000 (0.45s)
                --- PASS: TestMeta/execution/random-008 (2.28s)
                --- PASS: TestMeta/execution/standard-023 (0.34s)
                --- PASS: TestMeta/execution/standard-012 (0.38s)
                --- PASS: TestMeta/execution/standard-022 (0.34s)
                --- PASS: TestMeta/execution/standard-019 (0.24s)
                --- PASS: TestMeta/execution/standard-020 (0.27s)
                --- PASS: TestMeta/execution/random-012 (3.69s)
                --- PASS: TestMeta/execution/standard-018 (0.30s)
                --- PASS: TestMeta/execution/standard-017 (0.29s)
                --- PASS: TestMeta/execution/standard-016 (0.27s)
                --- PASS: TestMeta/execution/standard-014 (0.29s)
                --- PASS: TestMeta/execution/standard-013 (0.33s)
                --- PASS: TestMeta/execution/standard-006 (0.27s)
                --- PASS: TestMeta/execution/standard-011 (0.28s)
                --- PASS: TestMeta/execution/standard-010 (0.30s)
                --- PASS: TestMeta/execution/standard-009 (0.29s)
                --- PASS: TestMeta/execution/standard-007 (0.32s)
                --- PASS: TestMeta/execution/standard-003 (0.33s)
                --- PASS: TestMeta/execution/standard-004 (0.32s)
                --- PASS: TestMeta/execution/standard-002 (0.31s)
                --- PASS: TestMeta/execution/standard-021 (1.16s)
                --- PASS: TestMeta/execution/standard-008 (0.47s)
                --- PASS: TestMeta/execution/standard-015 (0.92s)
                --- PASS: TestMeta/execution/standard-005 (0.44s)
                --- PASS: TestMeta/execution/standard-001 (0.20s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-master/sha=c4d8b2a0-seed=16535223871199707113-opts=random-009(sha=837b5c6f-seed=8886979493832811155-opts=random-014(sha=4ddacdaf-seed=12224079438979341881-opts=random-019(<empty>))) (7.49s)
=== RUN   TestMetaCrossVersion/execution-master
    --- FAIL: TestMetaCrossVersion/execution-master (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7655496?buildTab=log) with [artifacts](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7655496?buildTab=artifacts#/) on refs/heads/master @ [936e011bb911](https://github.com/cockroachdb/pebble/commits/936e011bb911):


```
        snap5.Close()
        snap6.Close()
        snap8.Close()
        snap9.Close()
        db.Close()
        
        ===== HISTORY =====
        err: open /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2403352863/001/1669185473848175962/c4d8b2a0_5870856996365423013_001/_meta/221123-063813.8591340402859/standard-021/history: no such file or directory
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=random-017(sha=4ddacdaf-seed=5021791006041000601-opts=standard-015(<empty>)) (5.96s)
=== RUN   TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-020(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
=== PAUSE TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-020(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-020(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-020(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
    crossversion_test.go:223:   Running test with version c4d8b2a0 with initial state sha=837b5c6f-seed=9678769858472642518-opts=standard-020(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>)).
/test-bin/c4d8b2a020ea.test -test.run TestMeta$ -seed 5870856996365423013 -keep -ops uniform:1-10000 -dir /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2403352863/001/1669185473848175962/c4d8b2a0_5870856996365423013_003/_meta -test.v --initial-state /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2403352863/001/1669185473848175962/837b5c6f_9678769858472642518_006/_meta/221123-063809.7803520329019/standard-020 --initial-state-desc sha=837b5c6f-seed=9678769858472642518-opts=standard-020(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-020(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
    crossversion_test.go:267: Moving test dir "/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2403352863/001/1669185473848175962" to "/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/1669185473848175962".
    crossversion_test.go:268: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test.go:268
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test.go:226
        	Error:      	Received unexpected error:
        	            	rename /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2403352863/001/1669185473848175962 /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/1669185473848175962: file exists
        	Test:       	TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-020(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
        
        ===== HISTORY =====
        err: open /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2403352863/001/1669185473848175962/c4d8b2a0_5870856996365423013_004/_meta/221123-063819.6291291187095/random-021/history: no such file or directory
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-020(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>)) (5.91s)
=== RUN   TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=random-011(sha=4ddacdaf-seed=5021791006041000601-opts=random-012(<empty>))
=== PAUSE TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=random-011(sha=4ddacdaf-seed=5021791006041000601-opts=random-012(<empty>))
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=random-011(sha=4ddacdaf-seed=5021791006041000601-opts=random-012(<empty>))
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=random-011(sha=4ddacdaf-seed=5021791006041000601-opts=random-012(<empty>))
    crossversion_test.go:223:   Running test with version c4d8b2a0 with initial state sha=837b5c6f-seed=9678769858472642518-opts=random-011(sha=4ddacdaf-seed=5021791006041000601-opts=random-012(<empty>)).
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=random-011(sha=4ddacdaf-seed=5021791006041000601-opts=random-012(<empty>))
    crossversion_test.go:267: Moving test dir "/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2403352863/001/1669185473848175962" to "/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/1669185473848175962".
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=random-011(sha=4ddacdaf-seed=5021791006041000601-opts=random-012(<empty>)) (5.87s)
=== RUN   TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-009(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
=== PAUSE TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-009(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-009(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-009(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
    crossversion_test.go:223:   Running test with version c4d8b2a0 with initial state sha=837b5c6f-seed=9678769858472642518-opts=standard-009(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>)).
/test-bin/c4d8b2a020ea.test -test.run TestMeta$ -seed 5870856996365423013 -keep -ops uniform:1-10000 -dir /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2403352863/001/1669185473848175962/c4d8b2a0_5870856996365423013_007/_meta -test.v --initial-state /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2403352863/001/1669185473848175962/837b5c6f_9678769858472642518_006/_meta/221123-063809.7803520329019/standard-009 --initial-state-desc sha=837b5c6f-seed=9678769858472642518-opts=standard-009(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-009(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
    crossversion_test.go:267: Moving test dir "/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2403352863/001/1669185473848175962" to "/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/1669185473848175962".
    crossversion_test.go:268: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test.go:268
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test.go:226
        	Error:      	Received unexpected error:
        	            	rename /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2403352863/001/1669185473848175962 /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/1669185473848175962: file exists
        	Test:       	TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-009(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>))
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9678769858472642518-opts=standard-009(sha=4ddacdaf-seed=5021791006041000601-opts=standard-021(<empty>)) (5.91s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7685356?buildTab=log) with [artifacts](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7685356?buildTab=artifacts#/) on refs/heads/master @ [936e011bb911](https://github.com/cockroachdb/pebble/commits/936e011bb911):


```
db.Get("niza@4")
db.Get("grxlcnpyv@8")
db.Merge("oeba@1", "jxeq")
db.Delete("wgdbpoc@1")
db.SingleDelete("uruef@4", true /* maybeReplaceDelete */)
db.Merge("yzfmm@10", "weitalqhwhp")
db.Set("dgjbzitdoj@11", "qycbrkhcbrmzz")
db.Get("dsxdmdu@5")
db.DeleteRange("fjqljx", "pfpexj@7")
db.Flush()
db.Set("xgnjvavpzgy@1", "lbnruofdvozomp")
db.Delete("zublk@6")
db.Delete("azwpuojnb@3")
db.Delete("tkhsrouemo@6")
db.Get("ztqvbxjaqym")
db.Set("aeipud@3", "pecclppsblgjxfm")
db.Get("ojpohovbo@6")
db.Merge("fjqljx", "zm")
db.Get("wbooiv@8")
db.Delete("oybknxtxth")
db.Merge("eiknoqyu", "tgnwrjav")
db.Delete("awwqrlwkx@1")
db.SingleDelete("spjv@2", true /* maybeReplaceDelete */)
db.Merge("awwqrlwkx@4", "bdcnppfdszaehzev")
db.Delete("vofemlwp@6")
db.Merge("htwkrlyrmmq@9", "ldqldagtjvx")
db.Delete("fnyixhl@2")
db.DeleteRange("nvifbj@1", "totawegyfgs@7")
db.Merge("onvxkd", "johutvkvkgaotlpunsc")
db.Delete("ypywl@11")
db.Get("lifjkucqa@7")
db.Get("wyzvlgyg@8")
db.Delete("jwvrpiau@1")
db.Set("zdrgslst@7", "ctmuhukkwnbkul")
db.Get("pognukesv@1")
db.Set("osww@8", "vv")
db.SingleDelete("aeipud@3", false /* maybeReplaceDelete */)
db.Set("edxfdyzyi@1", "ninyjluqpncqbzqcs")
db.Merge("bohjzya@5", "rs")
db.Set("bgagt@9", "uybonbcy")
db.Get("zmrvas@4")
db.SingleDelete("dfmbskirx@10", true /* maybeReplaceDelete */)
db.DeleteRange("cvgjgiwkkvy@6", "oeba@1")
db.Set("tqgz@9", "rrrhduqsxumxutx")
db.Merge("mjptrntwig@6", "b")
db.Get("lqixl@4")
db.Close()

		=== RUN   TestMeta/compare/random-024
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=12131916391173137777-opts=random-016(sha=4ddacdaf-seed=17980092658089582826-opts=random-016(<empty>)) (12.71s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7709042?buildTab=log) with [artifacts](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7709042?buildTab=artifacts#/) on refs/heads/master @ [936e011bb911](https://github.com/cockroachdb/pebble/commits/936e011bb911):


```
        iter41.SeekGE("tbgyefqold@8", "")
        iter41.Prev("")
        iter41.Next("")
        db.SingleDelete("yazri@2", false /* maybeReplaceDelete */)
        db.Set("ebcxxtfm@8", "ysfepodutnc")
        iter41.Next("")
        iter41.SeekPrefixGE("tbgyefqold@8")
        iter41.SeekGE("bjtwxtk@11", "")
        db.Get("iatlgwuwq@5")
        iter41.SeekLT("ehfg@9", "")
        db.Merge("juhrrrimz@9", "iexiqhuz")
        db.Delete("aruzhfqe@2")
        iter41.Prev("")
        db.Delete("ynxticjveoe@7")
        db.Set("iocunbgpsyn@11", "esdthxelfrsspoz")
        iter41.SeekLT("hzwqvmgei@8", "")
        db.Get("jkvsj@8")
        iter41.First()
        iter41.First()
        iter41.Next("djhe@5")
        iter41.Last()
        iter41.First()
        iter41.SeekGE("ercmaiprigu@1", "xamx@9")
        iter41.SetBounds("dzddeprerg@8", "jiwb@11")
        iter41.SeekLT("jiwb@11", "")
        iter41.Next("")
        iter41.Prev("")
        iter41.First()
        db.RangeKeyDelete("aarx", "flhgyih")
        iter41.Last()
        iter41.First()
        db.Get("cmerkjicxd@1")
        iter41.SeekGE("yssldlrg@1", "")
        db.Delete("qyqxsotvyr@7")
        db.Merge("hkbbbuju@7", "nqmhhkwbcwwgbgxja")
        iter41.SeekPrefixGE("ehfg@6")
        iter41.First()
        iter41.SeekGE("ojppoig@6", "tyrvqgb@9")
        iter41.Next("")
        iter41.Prev("")
        iter41.Next("")
        iter41.SeekPrefixGE("ebcxxtfm@8")
       === RUN   TestMeta/compare
    history.go:181: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/history.go:181
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/meta_test.go:462
        	Error:      	Received unexpected error:
        	            	open /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion1221253901/001/1669529418767601517/936e011b_16269095637215081618_004/_meta/221127-061055.3992923560120/standard-000/history: no such file or directory
        	Test:       	TestMeta/compare
        --- FAIL: TestMetaCrossVersion/execution-master/sha=c4d8b2a0-seed=10661534282949991610-opts=standard-015(sha=837b5c6f-seed=1055945569208105149-opts=standard-016(sha=4ddacdaf-seed=15570355280614537231-opts=random-017(<empty>))) (22.36s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7815026?buildTab=log) with [artifacts](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7815026?buildTab=artifacts#/) on refs/heads/master @ [0fd6d4020f87](https://github.com/cockroachdb/pebble/commits/0fd6d4020f87):


```
=== CONT  TestMetaCrossVersion
    crossversion_test.go:157: Running tests with version c4d8b2a0 with 10 initial state(s).
=== CONT  TestMetaCrossVersion
    history.go:181: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/history.go:181
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/history.go:128
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test.go:166
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test.go:105
        	Error:      	Received unexpected error:
        	            	open /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion3123663030/001/1670062135856819088/c4d8b2a0_9331996466120092692_005/_meta/221203-100905.9211827737865/random-000/history: no such file or directory
        	Test:       	TestMetaCrossVersion
--- FAIL: TestMetaCrossVersion (11.97s)
=== RUN   TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=random-001(sha=4ddacdaf-seed=9887486739961441100-opts=random-021(<empty>))
=== PAUSE TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=random-001(sha=4ddacdaf-seed=9887486739961441100-opts=random-021(<empty>))
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=random-001(sha=4ddacdaf-seed=9887486739961441100-opts=random-021(<empty>))
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=random-001(sha=4ddacdaf-seed=9887486739961441100-opts=random-021(<empty>))
    crossversion_test.go:223:   Running test with version c4d8b2a0 with initial state sha=837b5c6f-seed=9237663335059761741-opts=random-001(sha=4ddacdaf-seed=9887486739961441100-opts=random-021(<empty>)).
/test-bin/c4d8b2a020ea.test -test.run TestMeta$ -seed 9331996466120092692 -keep -ops uniform:1-10000 -dir /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion3123663030/001/1670062135856819088/c4d8b2a0_9331996466120092692_007/_meta -test.v --initial-state /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion3123663030/001/1670062135856819088/837b5c6f_9237663335059761741_009/_meta/221203-100858.6232760634967/standard-004 --initial-state-desc sha=837b5c6f-seed=9237663335059761741-opts=standard-004(sha=4ddacdaf-seed=9887486739961441100-opts=random-014(<empty>))
/test-bin/c4d8b2a020ea.test -test.run TestMeta$ -seed 9331996466120092692 -keep -ops uniform:1-10000 -dir /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion3123663030/001/1670062135856819088/c4d8b2a0_9331996466120092692_006/_meta -test.v --initial-state /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion3123663030/001/1670062135856819088/837b5c6f_9237663335059761741_004/_meta/221203-100858.622767255684/random-001 --initial-state-desc sha=837b5c6f-seed=9237663335059761741-opts=random-001(sha=4ddacdaf-seed=9887486739961441100-opts=random-021(<empty>))
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=random-001(sha=4ddacdaf-seed=9887486739961441100-opts=random-021(<empty>))
    crossversion_test.go:267: Moving test dir "/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion3123663030/001/1670062135856819088" to "/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/1670062135856819088".
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=random-001(sha=4ddacdaf-seed=9887486739961441100-opts=random-021(<empty>)) (1.90s)
=== RUN   TestMetaCrossVersion/execution-crl-release-22.2
    --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2 (0.00s)
=== RUN   TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=standard-002(sha=4ddacdaf-seed=9887486739961441100-opts=standard-015(<empty>))
=== PAUSE TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=standard-002(sha=4ddacdaf-seed=9887486739961441100-opts=standard-015(<empty>))
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=standard-002(sha=4ddacdaf-seed=9887486739961441100-opts=standard-015(<empty>))
    crossversion_test.go:223:   Running test with version c4d8b2a0 with initial state sha=837b5c6f-seed=9237663335059761741-opts=standard-002(sha=4ddacdaf-seed=9887486739961441100-opts=standard-015(<empty>)).
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=standard-002(sha=4ddacdaf-seed=9887486739961441100-opts=standard-015(<empty>))
    crossversion_test.go:267: Moving test dir "/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion3123663030/001/1670062135856819088" to "/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/1670062135856819088".
    crossversion_test.go:268: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test.go:268
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test.go:226
        	Error:      	Received unexpected error:
        	            	rename /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion3123663030/001/1670062135856819088 /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/1670062135856819088: file exists
        	Test:       	TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=standard-002(sha=4ddacdaf-seed=9887486739961441100-opts=standard-015(<empty>))
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=standard-002(sha=4ddacdaf-seed=9887486739961441100-opts=standard-015(<empty>)) (0.52s)
=== RUN   TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=standard-007(sha=4ddacdaf-seed=9887486739961441100-opts=random-011(<empty>))
=== PAUSE TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=standard-007(sha=4ddacdaf-seed=9887486739961441100-opts=random-011(<empty>))
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=standard-007(sha=4ddacdaf-seed=9887486739961441100-opts=random-011(<empty>))
    crossversion_test.go:223:   Running test with version c4d8b2a0 with initial state sha=837b5c6f-seed=9237663335059761741-opts=standard-007(sha=4ddacdaf-seed=9887486739961441100-opts=random-011(<empty>)).
=== CONT  TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=standard-007(sha=4ddacdaf-seed=9887486739961441100-opts=random-011(<empty>))
    crossversion_test.go:267: Moving test dir "/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion3123663030/001/1670062135856819088" to "/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/1670062135856819088".
    history.go:181: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/history.go:181
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/meta_test.go:467
        	Error:      	Received unexpected error:
        	            	open /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion3123663030/001/1670062135856819088/c4d8b2a0_9331996466120092692_004/_meta/221203-100907.3073303964416/random-017/history: no such file or directory
        	Test:       	TestMeta/compare/random-017
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=9237663335059761741-opts=standard-007(sha=4ddacdaf-seed=9887486739961441100-opts=random-011(<empty>)) (0.48s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7824970?buildTab=log) with [artifacts](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7824970?buildTab=artifacts#/) on refs/heads/master @ [0fd6d4020f87](https://github.com/cockroachdb/pebble/commits/0fd6d4020f87):


```
        db.SingleDelete("svadhxto@12", false /* maybeReplaceDelete */)
        db.SingleDelete("nkwmjf@6", false /* maybeReplaceDelete */)
        db.Set("obqiu@6", "tjyavnql")
        db.Merge("yjcv@12", "ncijgwoxulqi")
        db.DeleteRange("fpzeykxznrc@10", "ghoqppxamf@5")
        db.Get("jexa@11")
        db.SingleDelete("pwlxsyeo@7", false /* maybeReplaceDelete */)
        db.Set("woiryhm@7", "iac")
        db.Merge("gevi@12", "s")
        db.SingleDelete("gsyexb@8", false /* maybeReplaceDelete */)
        db.Merge("hkvrphkvxsf@7", "ynttkinese")
        db.Set("fsenhuy@7", "chjjtvbeqkz")
        db.Delete("fssaz@12")
        db.DeleteRange("tngiewf", "vimaiccxu@3")
        db.Set("cotljdn@2", "dtcowxae")
        db.Get("jqxttrm@1")
        db.SingleDelete("ddglhnlptb@13", false /* maybeReplaceDelete */)
        db.Get("zscfj@6")
        db.DeleteRange("oljj@8", "xwwhga@3")
        db.Set("vbgja@5", "sjvmxforpdmjjxz")
        db.Merge("emsnvlebsxp@7", "fioxlefmuedlpyyd")
        db.Get("zkxfduikmt@7")
        db.Set("xitwe@8", "mbo")
        db.Delete("tkeoclnexvi@5")
        db.Get("ecljznf@2")
        db.DeleteRange("erqpllxty@1", "fssaz@9")
        db.RangeKeySet("ykjzo", "zkxfduikmt", "@1", "zqlerdcbs")
        db.Get("cxkr@2")
        db.Merge("wgswz@13", "hfqallvizeszzcohebb")
        db.Delete("jqxttrm@1")
        snap55 = db.NewSnapshot()
        db.Set("yakudqfny@6", "vtrccarswbfp")
        db.Delete("nfmtuslotx@9")
        db.Get("mqyozic@4")
        db.Merge("gtdq", "fgczdlaovakdptffy")
        snap55.Get("nibcfag@8")
        snap55.Get("vimaiccxu@7")
        db.RangeKeyUnset("apjz", "fpzeykxznrc", "@8")
        db.RangeKeySet("aaromwkljqx", "gsyexb", "@12", "vlza")
        snap55.Get("zdjzgvjvkxz@9")
        db.Set("roculr@4", "mwczyts")
        db.Merge("gtdq", "ulierfueax")
        db.Get("xwwhga@7")
        db.Get("qbmjfmavh@8")
        snap55.Close()
        db.Close()
        
        ===== HISTORY =====
        err: open /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion15504168/001/1670137932654000412/0fd6d402_8707321963528362052_006/_meta/221204-071426.2921553597629/standard-012/history: no such file or directory
        --- FAIL: TestMetaCrossVersion/execution-master/sha=c4d8b2a0-seed=8905917424881986068-opts=standard-002(sha=837b5c6f-seed=6688866557156242622-opts=random-016(sha=4ddacdaf-seed=1050029021649604788-opts=random-000(<empty>))) (4.16s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7834198?buildTab=log) with [artifacts](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7834198?buildTab=artifacts#/) on refs/heads/master @ [0fd6d4020f87](https://github.com/cockroachdb/pebble/commits/0fd6d4020f87):


```
        db.SingleDelete("seddk@9", false /* maybeReplaceDelete */)
        iter25.SeekLT("kezxyr@9", "")
        iter25.SeekLT("acyjsqmmpqd@7", "")
        db.DeleteRange("gpitxdhgbt@1", "gxfpefna@5")
        db.DeleteRange("mayzfb@3", "qyddwzdhlay@9")
        iter22.SeekGE("odihs@3", "")
        snap19.Get("vwzsty")
        db.SingleDelete("vffpewfqgt@8", false /* maybeReplaceDelete */)
        db.SingleDelete("kezxyr@7", true /* maybeReplaceDelete */)
        iter25.SeekPrefixGE("gsotcxkv@7")
        iter25.SeekGE("rpltrvrp@8", "")
        iter25.SeekGE("vweqymese@3", "")
        iter22.Last()
        iter22.SetBounds("azzhirif@9", "lazslqfusg@4")
        iter22.SeekLT("lazslqfusg@4", "")
        iter22.Prev("")
        iter25.Prev("")
        iter25.SeekLT("ztuewt@6", "")
        db.DeleteRange("vffpewfqgt@6", "zyarzex@9")
        snap19.Get("azzhirif@9")
        snap19.Get("fchaowmyf@8")
        db.Delete("qzekjr@5")
        iter25.First()
        iter25.SeekGE("jyvdzmly@7", "")
        iter25.SeekGE("vyotixicq@7", "")
        db.Merge("gclkljzqvm@5", "lmpcvcazbdvtngkv")
        db.DeleteRange("esfmxlldms", "mayzfb@2")
        iter22.Prev("")
        db.SingleDelete("fmrtqyo@6", false /* maybeReplaceDelete */)
        iter22.Next("")
        iter25.Next("")
        iter22.SeekGE("kezxyr@7", "")
        iter25.Next("")
        snap19.Get("xmeq@5")
        iter25.Next("")
        db.Merge("xxvkzvj@2", "wpwmqtxbaypqmq")
        db.Merge("vweqymese@3", "uwnhxnfnzcmdsuhow")
        db.RangeKeyUnset("ktis", "ooumhxdm", "@9")
        iter25.SeekLT("ncupka@3", "")
        iter25.Close()
        snap19.Close()
        iter22.SeekPrefixGE("knhpu@3")
        db.Get("vutrs@4")
        iter22.SeekLT("xoaqm@1", "")
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test.go:268
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test.go:226
        	Error:      	Received unexpected error:
        	            	rename /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2115511277/001/1670230748379168318 /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1021/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/1670230748379168318: file exists
        	Test:       	TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=15292723203688230726-opts=random-016(sha=4ddacdaf-seed=12821787456843760874-opts=standard-011(<empty>))
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=837b5c6f-seed=15292723203688230726-opts=random-016(sha=4ddacdaf-seed=12821787456843760874-opts=standard-011(<empty>)) (34.12s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7852035?buildTab=log) with [artifacts](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7852035?buildTab=artifacts#/) on refs/heads/master @ [4a63cdb3a71e](https://github.com/cockroachdb/pebble/commits/4a63cdb3a71e):


```
iter49.Next("")
iter49.Last()
db.Merge("rmcmocuvrb@10", "pzymguutck")
iter49.Last()
iter49.Last()
iter49.Prev("")
iter49.Next("")
iter49.SeekPrefixGE("rmcmocuvrb@10")
db.Get("qzeflzm@12")
iter49.Next("")
iter49.SeekGE("maxzkdfyqav@11", "")
db.Delete("fhkbbyveu@6")
db.Set("wbjmjsvjrc@9", "rvzy")
iter49.Prev("")
db.SingleDelete("gczd@18", false /* maybeReplaceDelete */)
db.Set("tmtqs@2", "fsaczjwackz")
iter49.Next("gczd@18")
iter49.Prev("")
iter49.Next("")
db.DeleteRange("qsva@12", "uzdul@6")
db.Merge("efay@12", "hxuetvysgqcx")
iter49.Prev("")
iter49.SeekLT("jxunkq@14", "jselnyjq@4")
iter49.SeekPrefixGE("qqhladyfin@11")
iter49.Prev("")
iter49.Close()
db.Delete("tmazka@7")
db.DeleteRange("hftekamvsi@11", "yuwspt@6")
db.RangeKeyUnset("sgol", "xsngo", "@18")
db.Merge("xllb@11", "jknoczznitgs")
db.Delete("fvmhyz@16")
db.RangeKeySet("jwtee", "opymfkbqrc", "@14", "m")
db.Delete("txtgknbexs")
db.Get("kfnngjhby@16")
db.Get("ytcpyfupna@9")
db.Set("brhvzten@9", "ifvrholerlu")
db.Set("lgng@12", "jakpwuklsscjifzbdp")
db.DeleteRange("keatokcfvy@13", "vxksommq@1")
db.DeleteRange("xugfkw@17", "zqgcxrht@10")
db.Set("etimq@16", "geuqbsteiptmwjuaiu")
db.SingleDelete("qbfyuncuna@16", false /* maybeReplaceDelete */)
db.Merge("egftudwp@10", "")
db.Get("fhkbbyveu@6")
db.Get("cbdkdf@17")
db.Close()

		=== CONT  TestMeta/execution/random-008
        --- FAIL: TestMetaCrossVersion/execution-master/sha=c4d8b2a0-seed=11175723037475337865-opts=random-002(sha=837b5c6f-seed=4015101241934861538-opts=random-008(sha=4ddacdaf-seed=9066947261237111487-opts=random-010(<empty>))) (403.19s)
=== RUN   TestMetaCrossVersion/execution-master
    --- FAIL: TestMetaCrossVersion/execution-master (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7909338?buildTab=log) with [artifacts](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7909338?buildTab=artifacts#/) on refs/heads/master @ [88c8fbcc3867](https://github.com/cockroachdb/pebble/commits/88c8fbcc3867):


```
                --- PASS: TestMeta/execution/standard-000 (3.39s)
                --- PASS: TestMeta/execution/random-000 (3.52s)
                --- PASS: TestMeta/execution/random-012 (3.73s)
                --- PASS: TestMeta/execution/random-018 (6.59s)
                --- PASS: TestMeta/execution/random-019 (3.77s)
                --- PASS: TestMeta/execution/random-006 (4.61s)
                --- PASS: TestMeta/execution/random-008 (3.70s)
                --- PASS: TestMeta/execution/random-007 (3.55s)
                --- PASS: TestMeta/execution/random-003 (3.69s)
                --- PASS: TestMeta/execution/random-011 (9.11s)
                --- PASS: TestMeta/execution/random-005 (3.64s)
                --- PASS: TestMeta/execution/random-015 (3.19s)
                --- PASS: TestMeta/execution/random-017 (3.10s)
                --- PASS: TestMeta/execution/random-016 (2.71s)
                --- PASS: TestMeta/execution/random-024 (16.76s)
                --- PASS: TestMeta/execution/random-014 (3.69s)
                --- PASS: TestMeta/execution/random-001 (4.19s)
                --- PASS: TestMeta/execution/standard-013 (2.84s)
                --- PASS: TestMeta/execution/random-002 (5.05s)
                --- PASS: TestMeta/execution/random-004 (12.66s)
                --- PASS: TestMeta/execution/standard-023 (2.92s)
                --- PASS: TestMeta/execution/standard-024 (3.44s)
                --- PASS: TestMeta/execution/standard-022 (2.89s)
                --- PASS: TestMeta/execution/random-020 (22.54s)
                --- PASS: TestMeta/execution/random-009 (22.48s)
                --- PASS: TestMeta/execution/standard-019 (1.95s)
                --- PASS: TestMeta/execution/standard-018 (1.58s)
                --- PASS: TestMeta/execution/random-010 (22.82s)
                --- PASS: TestMeta/execution/standard-020 (2.10s)
                --- PASS: TestMeta/execution/standard-017 (1.25s)
                --- PASS: TestMeta/execution/random-013 (11.15s)
                --- PASS: TestMeta/execution/standard-016 (0.79s)
                --- PASS: TestMeta/execution/standard-007 (0.86s)
                --- PASS: TestMeta/execution/standard-012 (0.86s)
                --- PASS: TestMeta/execution/standard-014 (1.02s)
                --- PASS: TestMeta/execution/standard-011 (1.04s)
                --- FAIL: TestMeta/execution/standard-005 (0.12s)
                --- PASS: TestMeta/execution/standard-021 (4.89s)
                --- PASS: TestMeta/execution/standard-010 (1.30s)
                --- PASS: TestMeta/execution/standard-009 (1.17s)
                --- PASS: TestMeta/execution/standard-006 (1.10s)
                --- PASS: TestMeta/execution/standard-004 (1.26s)
                --- PASS: TestMeta/execution/standard-002 (0.90s)
                --- PASS: TestMeta/execution/standard-015 (2.45s)
                --- PASS: TestMeta/execution/standard-008 (1.55s)
                --- PASS: TestMeta/execution/standard-001 (1.00s)
                --- PASS: TestMeta/execution/standard-003 (1.13s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=3a754874-seed=3679512399623515808-opts=standard-005(sha=4ddacdaf-seed=14457311065959589616-opts=random-004(<empty>)) (241.78s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

^
```
       meta_test.go:392: 
                ===== SEED =====
                6044308897238243864
                ===== ERR =====
                exit status 1
                ===== OUT =====
                --- FAIL: TestMeta (0.08s)
                    meta_test.go:202: 
                        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1369/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/meta_test.go:202
                        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1369/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test_/crossversion_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_pebble/internal/metamorphic/meta_test.go:307
                        	Error:      	Received unexpected error:
                        	            	migrating to version 10: open /artifacts/tmp/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2063945218/001/1670663076528809388/crl-release-22.2_0f086d93_6044308897238243864_004/_meta/221210-090446.535361931277/standard-005/data/000974.sst: file does not exist
                        	            	(1) attached stack trace
                        	            	  -- stack trace:
                        	            	  | github.com/cockroachdb/pebble.(*DB).ratchetFormatMajorVersionLocked
                        	            	  | 	github.com/cockroachdb/pebble/external/com_github_cockroachdb_pebble/format_major_version.go:352
                        	            	  | github.com/cockroachdb/pebble.Open
                        	            	  | 	github.com/cockroachdb/pebble/external/com_github_cockroachdb_pebble/open.go:268
                        	            	  | github.com/cockroachdb/pebble/internal/metamorphic.(*test).init.func12
                        	            	  | 	github.com/cockroachdb/pebble/internal/metamorphic/external/com_github_cockroachdb_pebble/internal/metamorphic/test.go:126
                        	            	  | github.com/cockroachdb/pebble/internal/metamorphic.withRetries
                        	            	  | 	github.com/cockroachdb/pebble/internal/metamorphic/external/com_github_cockroachdb_pebble/internal/metamorphic/retryable.go:21
                        	            	  | github.com/cockroachdb/pebble/internal/metamorphic.(*test).init
                        	            	  | 	github.com/cockroachdb/pebble/internal/metamorphic/external/com_github_cockroachdb_pebble/internal/metamorphic/test.go:125
                        	            	  | github.com/cockroachdb/pebble/internal/metamorphic.testMetaRun
                        	            	  | 	github.com/cockroachdb/pebble/internal/metamorphic/external/com_github_cockroachdb_pebble/internal/metamorphic/meta_test.go:202
                        	            	  | github.com/cockroachdb/pebble/internal/metamorphic.TestMeta
                        	            	  | 	github.com/cockroachdb/pebble/internal/metamorphic/external/com_github_cockroachdb_pebble/internal/metamorphic/meta_test.go:307
                        	            	  | testing.tRunner
                        	            	  | 	GOROOT/src/testing/testing.go:1446
                        	            	  | runtime.goexit
                        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
                        	            	Wraps: (2) migrating to version 10
                        	            	Wraps: (3) open /artifacts/tmp/_tmp/366cdd6e86440d4e9e9c98d28c5c6592/TestMetaCrossVersion2063945218/001/1670663076528809388/crl-release-22.2_0f086d93_6044308897238243864_004/_meta/221210-090446.535361931277/standard-005/data/000974.sst
                        	            	Wraps: (4) file does not exist
                        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *fs.PathError (4) *errors.errorString
                        	Test:       	TestMeta
                FAIL
                
```

maybe a race in the format major version migration? 10 is `FormatPrePebblev1Marked`.

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7936857?buildTab=log) with [artifacts](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7936857?buildTab=artifacts#/) on refs/heads/master @ [0893071d8a52](https://github.com/cockroachdb/pebble/commits/0893071d8a52):


```
        db.Merge("efix@15", "hxxxtdfvj")
        db.SingleDelete("qfqr@17", false /* maybeReplaceDelete */)
        db.Set("mpnbvjow@14", "nze")
        iter47.Last()
        iter45.SetBounds("fzpnryyp@6", "xcydwvqyp@15")
        iter45.SeekGE("fzpnryyp@6", "")
        iter45.Prev("")
        iter45.Next("")
        iter45.SeekGE("dobzshoxcfv@2", "nodlij@12")
        iter44.SeekGE("ktwzmkjjr@18", "lequibungke@2")
        db.Set("zpnxgbmrm@13", "ugx")
        iter47.SetBounds("ghvkkzgkdeh@3", "gnlrsjxj@12")
        iter47.SeekGE("ghvkkzgkdeh@3", "")
        iter47.Next("")
        iter46.SeekPrefixGE("lgaxurqh@1")
        iter45.SeekPrefixGE("wvyjajzfixe@10")
        iter45.SetOptions("fofjv@10", "oaedxsrcqba", 0 /* key types */, 0, 0, "" /* masking suffix */)
        iter45.SeekGE("pqoywyus@2", "xjwzh@11")
        db.Merge("enejz@1", "zmcngqwppgvuyjaktl")
        iter47.SetBounds("lgaxurqh@9", "nsabb@9")
        iter47.SeekGE("lgaxurqh@9", "")
        iter47.Prev("")
        iter47.Prev("")
        iter47.Next("")
        db.Delete("ntmysmrggu@7")
        iter43.SeekGE("wbou@3", "")
        iter47.Prev("")
        iter47.First()
        db.Merge("auoc@9", "")
        db.Delete("vuae@9")
        db.Get("cvypz@12")
        iter46.Next("pbwh@16")
        db.Get("kscad@16")
        db.Merge("jzlhyeder@5", "")
        iter46.Prev("")
        db.DeleteRange("wcno@12", "yewu@13")
        iter46.SeekLT("napnxi@10", "")
        iter45.Next("")
        iter43.SeekGE("vxulufvbdrw@12", "")
        db.Merge("tamoylf@13", "dqxmrhlpyepc")
        db.Delete("vuae@9")
        iter46.Close()
        iter44.Close()
        iter47.Close()
        iter45.Close()
        iter43.Close()
        db.Close()
        
        		
        --- FAIL: TestMetaCrossVersion/execution-master/sha=0f086d93-seed=5292265004406280964-opts=standard-002(sha=3a754874-seed=5167096424260890201-opts=random-005(sha=4ddacdaf-seed=16562485822311477170-opts=standard-001(<empty>))) (211.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7984651?buildTab=log) with [artifacts](https://buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7984651?buildTab=artifacts#/) on refs/heads/master @ [33377d59d70e](https://github.com/cockroachdb/pebble/commits/33377d59d70e):


```
                --- PASS: TestMeta/execution/random-020 (4.96s)
                --- PASS: TestMeta/execution/random-018 (4.78s)
                --- PASS: TestMeta/execution/random-021 (5.84s)
                --- PASS: TestMeta/execution/random-016 (3.18s)
                --- PASS: TestMeta/execution/random-023 (6.68s)
                --- PASS: TestMeta/execution/random-019 (6.81s)
                --- PASS: TestMeta/execution/random-015 (2.57s)
                --- PASS: TestMeta/execution/random-017 (5.85s)
                --- PASS: TestMeta/execution/random-013 (3.47s)
                --- PASS: TestMeta/execution/random-014 (4.77s)
                --- PASS: TestMeta/execution/random-012 (3.58s)
                --- PASS: TestMeta/execution/random-011 (6.45s)
                --- PASS: TestMeta/execution/random-006 (4.81s)
                --- PASS: TestMeta/execution/random-008 (6.00s)
                --- PASS: TestMeta/execution/random-005 (6.04s)
                --- PASS: TestMeta/execution/random-007 (7.53s)
                --- PASS: TestMeta/execution/random-004 (3.90s)
                --- PASS: TestMeta/execution/random-002 (3.86s)
                --- PASS: TestMeta/execution/random-003 (5.39s)
                --- PASS: TestMeta/execution/standard-013 (3.51s)
                --- PASS: TestMeta/execution/standard-023 (3.08s)
                --- PASS: TestMeta/execution/random-000 (22.14s)
                --- PASS: TestMeta/execution/standard-024 (5.64s)
                --- PASS: TestMeta/execution/standard-022 (3.66s)
                --- PASS: TestMeta/execution/standard-019 (2.42s)
                --- PASS: TestMeta/execution/standard-020 (3.43s)
                --- PASS: TestMeta/execution/standard-018 (3.76s)
                --- PASS: TestMeta/execution/standard-016 (3.29s)
                --- PASS: TestMeta/execution/standard-017 (4.25s)
                --- PASS: TestMeta/execution/standard-014 (3.54s)
                --- PASS: TestMeta/execution/standard-021 (9.67s)
                --- PASS: TestMeta/execution/standard-012 (2.47s)
                --- PASS: TestMeta/execution/random-010 (23.59s)
                --- PASS: TestMeta/execution/standard-007 (2.82s)
                --- PASS: TestMeta/execution/random-001 (15.38s)
                --- PASS: TestMeta/execution/standard-011 (1.86s)
                --- PASS: TestMeta/execution/standard-010 (2.00s)
                --- PASS: TestMeta/execution/random-009 (24.58s)
                --- PASS: TestMeta/execution/standard-009 (1.98s)
                --- PASS: TestMeta/execution/standard-004 (1.97s)
                --- PASS: TestMeta/execution/standard-015 (7.38s)
                --- PASS: TestMeta/execution/standard-006 (1.43s)
                --- PASS: TestMeta/execution/standard-002 (1.17s)
                --- PASS: TestMeta/execution/standard-008 (2.78s)
                --- PASS: TestMeta/execution/standard-003 (1.13s)
                --- PASS: TestMeta/execution/standard-001 (1.11s)
                --- PASS: TestMeta/execution/standard-005 (1.64s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=3a754874-seed=10148521667448672658-opts=random-017(sha=4ddacdaf-seed=1189054855832160165-opts=random-012(<empty>)) (190.13s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7999714?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/7999714?buildTab=artifacts#/) on refs/heads/master @ [33377d59d70e](https://github.com/cockroachdb/pebble/commits/33377d59d70e):


```
                --- PASS: TestMeta/execution/random-021 (0.49s)
                --- PASS: TestMeta/execution/random-020 (0.51s)
                --- PASS: TestMeta/execution/random-023 (0.66s)
                --- PASS: TestMeta/execution/random-018 (0.39s)
                --- PASS: TestMeta/execution/random-019 (0.71s)
                --- PASS: TestMeta/execution/random-014 (0.25s)
                --- PASS: TestMeta/execution/random-017 (0.47s)
                --- PASS: TestMeta/execution/random-022 (0.87s)
                --- PASS: TestMeta/execution/random-016 (0.48s)
                --- PASS: TestMeta/execution/standard-013 (0.33s)
                --- PASS: TestMeta/execution/standard-024 (0.30s)
                --- PASS: TestMeta/execution/standard-023 (0.36s)
                --- FAIL: TestMeta/execution/standard-018 (0.10s)
                --- PASS: TestMeta/execution/standard-022 (0.41s)
                --- PASS: TestMeta/execution/standard-020 (0.32s)
                --- PASS: TestMeta/execution/standard-019 (0.31s)
                --- PASS: TestMeta/execution/standard-017 (0.30s)
                --- PASS: TestMeta/execution/standard-016 (0.26s)
                --- PASS: TestMeta/execution/random-015 (0.85s)
                --- PASS: TestMeta/execution/standard-014 (0.23s)
                --- PASS: TestMeta/execution/standard-007 (0.28s)
                --- PASS: TestMeta/execution/standard-012 (0.27s)
                --- PASS: TestMeta/execution/standard-009 (0.24s)
                --- PASS: TestMeta/execution/standard-011 (0.28s)
                --- PASS: TestMeta/execution/standard-010 (0.36s)
                --- PASS: TestMeta/execution/standard-015 (0.72s)
                --- PASS: TestMeta/execution/standard-008 (0.46s)
                --- PASS: TestMeta/execution/random-012 (0.51s)
                --- PASS: TestMeta/execution/standard-021 (1.28s)
                --- PASS: TestMeta/execution/random-007 (0.64s)
                --- PASS: TestMeta/execution/random-004 (0.53s)
                --- PASS: TestMeta/execution/random-010 (0.81s)
                --- PASS: TestMeta/execution/random-009 (0.73s)
                --- PASS: TestMeta/execution/random-011 (0.89s)
                --- PASS: TestMeta/execution/random-003 (0.27s)
                --- PASS: TestMeta/execution/standard-004 (0.30s)
                --- PASS: TestMeta/execution/standard-006 (0.30s)
                --- PASS: TestMeta/execution/standard-005 (0.29s)
                --- PASS: TestMeta/execution/random-008 (0.98s)
                --- PASS: TestMeta/execution/random-005 (0.72s)
                --- PASS: TestMeta/execution/standard-002 (0.18s)
                --- PASS: TestMeta/execution/standard-003 (0.16s)
                --- PASS: TestMeta/execution/random-001 (0.15s)
                --- PASS: TestMeta/execution/standard-001 (0.19s)
                --- PASS: TestMeta/execution/random-000 (0.15s)
                --- PASS: TestMeta/execution/random-006 (0.95s)
                --- PASS: TestMeta/execution/random-002 (1.07s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=3a754874-seed=8554228005809998782-opts=random-020(sha=4ddacdaf-seed=10091272476951562545-opts=random-020(<empty>)) (129.76s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/8046011?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/8046011?buildTab=artifacts#/) on refs/heads/master @ [4510a8c5f244](https://github.com/cockroachdb/pebble/commits/4510a8c5f244):


```
                --- PASS: TestMeta/execution/standard-000 (0.82s)
                --- PASS: TestMeta/execution/random-024 (0.86s)
                --- PASS: TestMeta/execution/standard-013 (0.90s)
                --- PASS: TestMeta/execution/random-023 (1.32s)
                --- PASS: TestMeta/execution/random-020 (1.43s)
                --- PASS: TestMeta/execution/random-016 (1.19s)
                --- PASS: TestMeta/execution/random-017 (1.65s)
                --- PASS: TestMeta/execution/random-021 (1.94s)
                --- FAIL: TestMeta/execution/random-015 (0.84s)
                --- PASS: TestMeta/execution/random-000 (3.21s)
                --- PASS: TestMeta/execution/random-010 (0.78s)
                --- PASS: TestMeta/execution/random-013 (1.56s)
                --- PASS: TestMeta/execution/random-012 (1.56s)
                --- PASS: TestMeta/execution/random-018 (3.36s)
                --- PASS: TestMeta/execution/random-022 (4.00s)
                --- PASS: TestMeta/execution/random-011 (1.75s)
                --- PASS: TestMeta/execution/standard-016 (0.91s)
                --- PASS: TestMeta/execution/random-014 (2.46s)
                --- PASS: TestMeta/execution/standard-018 (1.05s)
                --- PASS: TestMeta/execution/random-009 (1.32s)
                --- PASS: TestMeta/execution/random-019 (4.87s)
                --- PASS: TestMeta/execution/standard-017 (1.31s)
                --- PASS: TestMeta/execution/random-006 (1.08s)
                --- PASS: TestMeta/execution/random-005 (0.83s)
                --- PASS: TestMeta/execution/random-004 (1.54s)
                --- PASS: TestMeta/execution/random-003 (1.71s)
                --- PASS: TestMeta/execution/random-007 (2.60s)
                --- PASS: TestMeta/execution/standard-015 (2.44s)
                --- PASS: TestMeta/execution/standard-022 (1.41s)
                --- PASS: TestMeta/execution/random-002 (1.59s)
                --- PASS: TestMeta/execution/random-001 (0.98s)
                --- PASS: TestMeta/execution/standard-020 (0.70s)
                --- PASS: TestMeta/execution/standard-012 (0.88s)
                --- PASS: TestMeta/execution/standard-006 (0.93s)
                --- PASS: TestMeta/execution/standard-007 (1.26s)
                --- PASS: TestMeta/execution/standard-004 (1.30s)
                --- PASS: TestMeta/execution/standard-011 (0.78s)
                --- PASS: TestMeta/execution/standard-010 (0.56s)
                --- PASS: TestMeta/execution/standard-009 (0.34s)
                --- PASS: TestMeta/execution/standard-005 (0.98s)
                --- PASS: TestMeta/execution/standard-003 (0.63s)
                --- PASS: TestMeta/execution/standard-002 (0.87s)
                --- PASS: TestMeta/execution/standard-008 (1.08s)
                --- PASS: TestMeta/execution/standard-001 (0.92s)
                --- PASS: TestMeta/execution/standard-014 (1.04s)
                --- PASS: TestMeta/execution/standard-021 (4.36s)
                --- PASS: TestMeta/execution/random-008 (6.54s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=3a754874-seed=4415541195790883972-opts=random-016(sha=4ddacdaf-seed=9158225543712878224-opts=random-000(<empty>)) (96.97s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/8087918?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/8087918?buildTab=artifacts#/) on refs/heads/master @ [eb5e1039627d](https://github.com/cockroachdb/pebble/commits/eb5e1039627d):


```
                --- PASS: TestMeta/execution/random-013 (1.71s)
                --- PASS: TestMeta/execution/standard-013 (1.74s)
                --- PASS: TestMeta/execution/random-021 (1.75s)
                --- PASS: TestMeta/execution/random-024 (2.14s)
                --- FAIL: TestMeta/execution/random-018 (0.79s)
                --- PASS: TestMeta/execution/random-020 (1.71s)
                --- PASS: TestMeta/execution/random-015 (1.50s)
                --- PASS: TestMeta/execution/random-017 (1.67s)
                --- PASS: TestMeta/execution/random-000 (3.63s)
                --- PASS: TestMeta/execution/standard-007 (1.66s)
                --- PASS: TestMeta/execution/standard-010 (0.73s)
                --- PASS: TestMeta/execution/standard-011 (0.88s)
                --- PASS: TestMeta/execution/standard-012 (1.33s)
                --- PASS: TestMeta/execution/random-019 (3.72s)
                --- PASS: TestMeta/execution/standard-005 (1.14s)
                --- PASS: TestMeta/execution/standard-006 (1.23s)
                --- PASS: TestMeta/execution/standard-009 (1.76s)
                --- PASS: TestMeta/execution/standard-004 (1.30s)
                --- PASS: TestMeta/execution/standard-008 (1.98s)
                --- PASS: TestMeta/execution/standard-002 (1.09s)
                --- PASS: TestMeta/execution/standard-019 (0.96s)
                --- PASS: TestMeta/execution/standard-003 (1.03s)
                --- PASS: TestMeta/execution/standard-023 (1.19s)
                --- PASS: TestMeta/execution/standard-024 (1.22s)
                --- PASS: TestMeta/execution/standard-022 (1.19s)
                --- PASS: TestMeta/execution/random-014 (4.93s)
                --- PASS: TestMeta/execution/standard-001 (0.90s)
                --- PASS: TestMeta/execution/standard-020 (1.08s)
                --- PASS: TestMeta/execution/random-016 (5.69s)
                --- PASS: TestMeta/execution/standard-018 (1.12s)
                --- PASS: TestMeta/execution/standard-016 (1.29s)
                --- PASS: TestMeta/execution/standard-014 (0.67s)
                --- PASS: TestMeta/execution/standard-017 (1.02s)
                --- PASS: TestMeta/execution/random-007 (0.94s)
                --- PASS: TestMeta/execution/standard-015 (1.36s)
                --- PASS: TestMeta/execution/random-012 (1.20s)
                --- PASS: TestMeta/execution/random-008 (0.63s)
                --- PASS: TestMeta/execution/random-010 (0.79s)
                --- PASS: TestMeta/execution/random-011 (1.01s)
                --- PASS: TestMeta/execution/random-009 (0.94s)
                --- PASS: TestMeta/execution/random-006 (0.43s)
                --- PASS: TestMeta/execution/random-004 (0.55s)
                --- PASS: TestMeta/execution/random-003 (0.24s)
                --- PASS: TestMeta/execution/random-002 (0.42s)
                --- PASS: TestMeta/execution/random-005 (0.76s)
                --- PASS: TestMeta/execution/standard-021 (3.51s)
                --- PASS: TestMeta/execution/random-001 (0.76s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=3a754874-seed=18192391513084261480-opts=random-001(sha=4ddacdaf-seed=3933480583706930242-opts=random-017(<empty>)) (106.52s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/8223713?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/8223713?buildTab=artifacts#/) on refs/heads/master @ [65ff304d3d7a](https://github.com/cockroachdb/pebble/commits/65ff304d3d7a):


```
                --- PASS: TestMeta/execution/random-013 (6.10s)
                --- PASS: TestMeta/execution/standard-023 (3.24s)
                --- PASS: TestMeta/execution/standard-024 (4.70s)
                --- PASS: TestMeta/execution/standard-022 (4.24s)
                --- PASS: TestMeta/execution/standard-019 (2.41s)
                --- PASS: TestMeta/execution/standard-020 (3.95s)
                --- PASS: TestMeta/execution/standard-018 (2.71s)
                --- PASS: TestMeta/execution/standard-016 (3.46s)
                --- PASS: TestMeta/execution/standard-017 (3.70s)
                --- PASS: TestMeta/execution/random-019 (11.23s)
                --- PASS: TestMeta/execution/standard-014 (2.68s)
                --- PASS: TestMeta/execution/standard-006 (3.98s)
                --- PASS: TestMeta/execution/standard-004 (3.58s)
                --- PASS: TestMeta/execution/standard-003 (3.61s)
                --- PASS: TestMeta/execution/standard-001 (2.00s)
                --- PASS: TestMeta/execution/standard-002 (3.78s)
                --- PASS: TestMeta/execution/standard-005 (4.74s)
                --- PASS: TestMeta/execution/standard-021 (12.39s)
                --- PASS: TestMeta/execution/standard-010 (3.36s)
                --- PASS: TestMeta/execution/standard-012 (4.22s)
                --- PASS: TestMeta/execution/standard-011 (4.18s)
                --- PASS: TestMeta/execution/random-007 (2.68s)
                --- PASS: TestMeta/execution/standard-009 (3.47s)
                --- PASS: TestMeta/execution/standard-015 (12.26s)
                --- PASS: TestMeta/execution/standard-008 (5.29s)
                --- PASS: TestMeta/execution/random-020 (22.52s)
                --- PASS: TestMeta/execution/random-012 (4.32s)
                --- PASS: TestMeta/execution/random-008 (4.51s)
                --- PASS: TestMeta/execution/random-010 (5.05s)
                --- PASS: TestMeta/execution/random-009 (5.89s)
                --- PASS: TestMeta/execution/random-023 (4.38s)
                --- PASS: TestMeta/execution/random-025 (4.24s)
                --- PASS: TestMeta/execution/random-004 (4.86s)
                --- PASS: TestMeta/execution/random-024 (6.75s)
                --- PASS: TestMeta/execution/random-016 (3.98s)
                --- PASS: TestMeta/execution/random-005 (6.45s)
                --- PASS: TestMeta/execution/random-002 (7.40s)
                --- PASS: TestMeta/execution/random-018 (4.82s)
                --- PASS: TestMeta/execution/random-017 (5.57s)
                --- PASS: TestMeta/execution/random-022 (4.81s)
                --- PASS: TestMeta/execution/random-014 (3.06s)
                --- PASS: TestMeta/execution/random-003 (11.50s)
                --- PASS: TestMeta/execution/random-011 (17.83s)
                --- PASS: TestMeta/execution/random-001 (4.09s)
                --- PASS: TestMeta/execution/random-006 (16.61s)
                --- PASS: TestMeta/execution/random-021 (7.99s)
                --- FAIL: TestMeta/execution/random-015 (13.60s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-master/sha=abd074e2-seed=15189583690057957301-opts=random-024(sha=3a754874-seed=17744341652947402851-opts=standard-013(sha=4ddacdaf-seed=1499861871223320569-opts=standard-012(<empty>))) (338.36s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/8230865?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/8230865?buildTab=artifacts#/) on refs/heads/master @ [65ff304d3d7a](https://github.com/cockroachdb/pebble/commits/65ff304d3d7a):


```
                --- PASS: TestMeta/execution/standard-024 (3.53s)
                --- PASS: TestMeta/execution/standard-025 (3.69s)
                --- PASS: TestMeta/execution/standard-023 (3.83s)
                --- PASS: TestMeta/execution/standard-022 (3.65s)
                --- PASS: TestMeta/execution/standard-019 (2.16s)
                --- PASS: TestMeta/execution/standard-020 (3.71s)
                --- PASS: TestMeta/execution/standard-017 (3.43s)
                --- PASS: TestMeta/execution/standard-018 (3.80s)
                --- PASS: TestMeta/execution/standard-016 (4.69s)
                --- PASS: TestMeta/execution/standard-014 (3.64s)
                --- PASS: TestMeta/execution/random-020 (4.93s)
                --- PASS: TestMeta/execution/standard-021 (10.13s)
                --- PASS: TestMeta/execution/random-024 (4.16s)
                --- PASS: TestMeta/execution/random-008 (16.42s)
                --- PASS: TestMeta/execution/random-022 (4.48s)
                --- PASS: TestMeta/execution/random-006 (4.85s)
                --- PASS: TestMeta/execution/random-004 (5.68s)
                --- PASS: TestMeta/execution/random-005 (4.60s)
                --- PASS: TestMeta/execution/standard-015 (12.88s)
                --- PASS: TestMeta/execution/random-002 (4.38s)
                --- PASS: TestMeta/execution/random-025 (11.23s)
                --- PASS: TestMeta/execution/standard-011 (3.08s)
                --- PASS: TestMeta/execution/standard-012 (3.36s)
                --- PASS: TestMeta/execution/standard-007 (3.83s)
                --- PASS: TestMeta/execution/standard-009 (3.26s)
                --- PASS: TestMeta/execution/random-021 (11.02s)
                --- PASS: TestMeta/execution/random-023 (13.07s)
                --- PASS: TestMeta/execution/standard-010 (4.10s)
                --- PASS: TestMeta/execution/random-003 (6.15s)
                --- PASS: TestMeta/execution/standard-004 (2.86s)
                --- PASS: TestMeta/execution/standard-008 (3.17s)
                --- PASS: TestMeta/execution/standard-002 (2.18s)
                --- PASS: TestMeta/execution/standard-006 (2.48s)
                --- PASS: TestMeta/execution/standard-003 (2.69s)
                --- PASS: TestMeta/execution/standard-005 (3.31s)
                --- PASS: TestMeta/execution/standard-001 (1.77s)
                --- PASS: TestMeta/execution/random-017 (3.19s)
                --- PASS: TestMeta/execution/random-018 (2.29s)
                --- PASS: TestMeta/execution/random-015 (1.68s)
                --- PASS: TestMeta/execution/random-016 (1.48s)
                --- PASS: TestMeta/execution/random-011 (2.67s)
                --- PASS: TestMeta/execution/random-000 (2.54s)
                --- PASS: TestMeta/execution/random-001 (7.88s)
                --- PASS: TestMeta/execution/random-019 (6.77s)
                --- PASS: TestMeta/execution/random-014 (6.14s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-master/sha=abd074e2-seed=14508211929618645386-opts=random-018(sha=3a754874-seed=3593902223197027522-opts=random-005(sha=4ddacdaf-seed=16998111122741151206-opts=random-009(<empty>))) (366.36s)
=== RUN   TestMetaCrossVersion/execution-master
    --- FAIL: TestMetaCrossVersion/execution-master (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/8240101?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/8240101?buildTab=artifacts#/) on refs/heads/master @ [65ff304d3d7a](https://github.com/cockroachdb/pebble/commits/65ff304d3d7a):


```
                --- PASS: TestMeta/execution/random-023 (4.74s)
                --- PASS: TestMeta/execution/random-021 (6.26s)
                --- PASS: TestMeta/execution/random-017 (3.14s)
                --- PASS: TestMeta/execution/random-018 (3.45s)
                --- PASS: TestMeta/execution/random-016 (2.84s)
                --- PASS: TestMeta/execution/random-025 (7.09s)
                --- PASS: TestMeta/execution/random-020 (6.04s)
                --- PASS: TestMeta/execution/random-011 (2.74s)
                --- PASS: TestMeta/execution/random-015 (5.89s)
                --- PASS: TestMeta/execution/random-009 (3.33s)
                --- PASS: TestMeta/execution/random-010 (5.06s)
                --- PASS: TestMeta/execution/random-008 (2.70s)
                --- PASS: TestMeta/execution/random-004 (1.80s)
                --- PASS: TestMeta/execution/random-007 (6.51s)
                --- PASS: TestMeta/execution/random-006 (3.48s)
                --- PASS: TestMeta/execution/standard-025 (2.33s)
                --- PASS: TestMeta/execution/standard-013 (2.78s)
                --- PASS: TestMeta/execution/standard-024 (2.94s)
                --- PASS: TestMeta/execution/standard-023 (2.63s)
                --- PASS: TestMeta/execution/standard-022 (2.60s)
                --- PASS: TestMeta/execution/standard-020 (2.31s)
                --- PASS: TestMeta/execution/random-012 (11.76s)
                --- PASS: TestMeta/execution/standard-018 (1.70s)
                --- PASS: TestMeta/execution/standard-019 (2.20s)
                --- PASS: TestMeta/execution/standard-014 (1.30s)
                --- PASS: TestMeta/execution/standard-017 (2.78s)
                --- PASS: TestMeta/execution/standard-016 (2.83s)
                --- FAIL: TestMeta/execution/random-005 (10.58s)
                --- PASS: TestMeta/execution/random-002 (2.09s)
                --- PASS: TestMeta/execution/standard-007 (2.46s)
                --- PASS: TestMeta/execution/standard-015 (6.22s)
                --- PASS: TestMeta/execution/standard-021 (10.55s)
                --- PASS: TestMeta/execution/standard-012 (3.64s)
                --- PASS: TestMeta/execution/standard-011 (2.23s)
                --- PASS: TestMeta/execution/random-019 (24.16s)
                --- PASS: TestMeta/execution/standard-010 (2.71s)
                --- PASS: TestMeta/execution/standard-009 (2.77s)
                --- PASS: TestMeta/execution/random-014 (22.40s)
                --- PASS: TestMeta/execution/standard-006 (2.34s)
                --- PASS: TestMeta/execution/random-003 (8.60s)
                --- PASS: TestMeta/execution/standard-004 (2.47s)
                --- PASS: TestMeta/execution/standard-008 (3.52s)
                --- PASS: TestMeta/execution/standard-005 (2.32s)
                --- PASS: TestMeta/execution/standard-002 (2.11s)
                --- PASS: TestMeta/execution/standard-003 (1.88s)
                --- PASS: TestMeta/execution/random-001 (9.29s)
                --- PASS: TestMeta/execution/standard-001 (1.29s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-master/sha=abd074e2-seed=3448051722486352910-opts=standard-000(sha=3a754874-seed=18086911377321110457-opts=random-012(sha=4ddacdaf-seed=9617010300008281683-opts=standard-008(<empty>))) (239.29s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/8286055?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/8286055?buildTab=artifacts#/) on refs/heads/master @ [5bad2c1a77ab](https://github.com/cockroachdb/pebble/commits/5bad2c1a77ab):


```
                --- PASS: TestMeta/execution/random-005 (4.79s)
                --- PASS: TestMeta/execution/random-018 (4.90s)
                --- PASS: TestMeta/execution/random-004 (2.98s)
                --- PASS: TestMeta/execution/random-003 (4.21s)
                --- PASS: TestMeta/execution/random-006 (8.13s)
                --- PASS: TestMeta/execution/random-007 (3.53s)
                --- PASS: TestMeta/execution/random-000 (9.72s)
                --- PASS: TestMeta/execution/random-001 (5.50s)
                --- PASS: TestMeta/execution/random-024 (11.96s)
                --- PASS: TestMeta/execution/random-021 (4.32s)
                --- PASS: TestMeta/execution/random-008 (8.88s)
                --- PASS: TestMeta/execution/random-017 (5.04s)
                --- PASS: TestMeta/execution/random-016 (4.89s)
                --- PASS: TestMeta/execution/random-013 (4.06s)
                --- PASS: TestMeta/execution/standard-013 (4.05s)
                --- PASS: TestMeta/execution/random-022 (6.97s)
                --- PASS: TestMeta/execution/standard-023 (3.33s)
                --- FAIL: TestMeta/execution/standard-018 (1.21s)
                --- PASS: TestMeta/execution/standard-024 (4.69s)
                --- PASS: TestMeta/execution/standard-022 (3.48s)
                --- PASS: TestMeta/execution/standard-020 (3.73s)
                --- PASS: TestMeta/execution/random-002 (16.21s)
                --- PASS: TestMeta/execution/standard-019 (3.63s)
                --- PASS: TestMeta/execution/standard-016 (3.39s)
                --- PASS: TestMeta/execution/standard-017 (4.13s)
                --- PASS: TestMeta/execution/standard-014 (3.56s)
                --- PASS: TestMeta/execution/random-015 (3.49s)
                --- PASS: TestMeta/execution/random-011 (5.26s)
                --- PASS: TestMeta/execution/random-020 (4.06s)
                --- PASS: TestMeta/execution/standard-007 (3.11s)
                --- PASS: TestMeta/execution/standard-012 (3.36s)
                --- PASS: TestMeta/execution/standard-010 (3.32s)
                --- PASS: TestMeta/execution/random-023 (20.85s)
                --- PASS: TestMeta/execution/standard-009 (2.62s)
                --- PASS: TestMeta/execution/standard-015 (10.62s)
                --- PASS: TestMeta/execution/standard-011 (2.76s)
                --- PASS: TestMeta/execution/random-014 (8.10s)
                --- PASS: TestMeta/execution/standard-021 (14.39s)
                --- PASS: TestMeta/execution/standard-006 (1.64s)
                --- PASS: TestMeta/execution/random-010 (2.02s)
                --- PASS: TestMeta/execution/standard-008 (4.01s)
                --- PASS: TestMeta/execution/random-019 (1.99s)
                --- PASS: TestMeta/execution/standard-005 (1.63s)
                --- PASS: TestMeta/execution/standard-002 (1.11s)
                --- PASS: TestMeta/execution/standard-004 (1.87s)
                --- PASS: TestMeta/execution/standard-003 (0.92s)
                --- PASS: TestMeta/execution/standard-001 (0.57s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-22.2/sha=3a754874-seed=11864772633194996220-opts=random-017(sha=4ddacdaf-seed=10708886204986670701-opts=standard-012(<empty>)) (312.52s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/format_major_version.go
+++ b/format_major_version.go
@@ -521,31 +521,24 @@ func (d *DB) markFilesLocked(findFn findFilesFunc) error {
 	jobID := d.mu.nextJobID
 	d.mu.nextJobID++
 
-	vers := d.mu.versions.currentVersion()
+	// Acquire a read state to have a view of the LSM and a guarantee that none
+	// of the referenced files will be deleted until we've unreferenced the read
+	// state. Some findFilesFuncs may read the files, requiring they not be
+	// deleted.
+	rs := d.loadReadState()
 	var (
 		found bool
 		files [numLevels][]*fileMetadata
 		err   error
 	)
 	func() {
-		// Disable file deletions. Some migrations that mark files may need to
-		// read the files themselves. The LSM may be changing while the
-		// migration is determining which files to mark. That's ok, because
-		// after re-acquiring the mutex and acquiring the manifest lock, we'll
-		// discard any sstables that have been removed from the LSM. However, we
-		// also need to make sure that the files we try to open during the scan
-		// are not removed before we read them. This is preferable over
-		// gracefully handling nonexistent files because some environments (eg,
-		// Windows) error if you attempt to remove an open file.
-		d.disableFileDeletions()
-		defer d.enableFileDeletions()
-
+		defer rs.unrefLocked()
 		// Note the unusual locking: unlock, defer Lock(). The scan of the files in
 		// the version does not need to block other operations that require the
 		// DB.mu. Drop it for the scan, before re-acquiring it.
 		d.mu.Unlock()
 		defer d.mu.Lock()
-		found, files, err = findFn(vers)
+		found, files, err = findFn(rs.current)
 	}()
 	if err != nil {
 		return err
@@ -566,7 +559,7 @@ func (d *DB) markFilesLocked(findFn findFilesFunc) error {
 	// Lock the manifest for a coherent view of the LSM. The database lock has
 	// been re-acquired by the defer within the above anonymous function.
 	d.mu.versions.logLock()
-	vers = d.mu.versions.currentVersion()
+	vers := d.mu.versions.currentVersion()
 	for l, filesToMark := range files {
 		if len(filesToMark) == 0 {
 			continue

--- a/open.go
+++ b/open.go
@@ -260,16 +260,6 @@ func Open(dirname string, opts *Options) (db *DB, _ error) {
 		}
 	}
 
-	// If the Options specify a format major version higher than the
-	// loaded database's, upgrade it. If this is a new database, this
-	// code path also performs an initial upgrade from the starting
-	// implicit MostCompatible version.
-	if !d.opts.ReadOnly && opts.FormatMajorVersion > d.mu.formatVers.vers {
-		if err := d.ratchetFormatMajorVersionLocked(opts.FormatMajorVersion); err != nil {
-			return nil, err
-		}
-	}
-
 	// Atomic markers like the one used for the MANIFEST may leave
 	// behind obsolete files if there's a crash mid-update. Clean these
 	// up if we're not in read-only mode.
@@ -432,6 +422,19 @@ func Open(dirname string, opts *Options) (db *DB, _ error) {
 		d.mu.versions.metrics.WAL.Files++
 	}
 	d.updateReadStateLocked(d.opts.DebugCheck)
+
+	// If the Options specify a format major version higher than the
+	// loaded database's, upgrade it. If this is a new database, this
+	// code path also performs an initial upgrade from the starting
+	// implicit MostCompatible version.
+	//
+	// We ratchet the version this far into Open so that migrations have a read
+	// state available.
+	if !d.opts.ReadOnly && opts.FormatMajorVersion > d.mu.formatVers.vers {
+		if err := d.ratchetFormatMajorVersionLocked(opts.FormatMajorVersion); err != nil {
+			return nil, err
+		}
+	}
 
 	if !d.opts.ReadOnly {
 		// Write the current options to disk.

--- a/testdata/checkpoint
+++ b/testdata/checkpoint
@@ -12,6 +12,9 @@ sync: db/temporary.000001.dbtmp
 close: db/temporary.000001.dbtmp
 rename: db/temporary.000001.dbtmp -> db/CURRENT
 sync: db
+sync: db/MANIFEST-000001
+create: db/000002.log
+sync: db
 create: db/marker.manifest.000001.MANIFEST-000001
 close: db/marker.manifest.000001.MANIFEST-000001
 sync: db
@@ -51,9 +54,6 @@ close: db/marker.format-version.000010.011
 sync: db
 create: db/marker.format-version.000011.012
 close: db/marker.format-version.000011.012
-sync: db
-sync: db/MANIFEST-000001
-create: db/000002.log
 sync: db
 create: db/temporary.000003.dbtmp
 sync: db/temporary.000003.dbtmp

--- a/testdata/event_listener
+++ b/testdata/event_listener
@@ -15,6 +15,10 @@ close: db/temporary.000001.dbtmp
 rename: db/temporary.000001.dbtmp -> db/CURRENT
 sync: db
 [JOB 1] MANIFEST created 000001
+sync: db/MANIFEST-000001
+create: wal/000002.log
+sync: wal
+[JOB 1] WAL created 000002
 create: db/marker.manifest.000001.MANIFEST-000001
 close: db/marker.manifest.000001.MANIFEST-000001
 sync: db
@@ -66,10 +70,6 @@ create: db/marker.format-version.000011.012
 close: db/marker.format-version.000011.012
 sync: db
 upgraded to format version: 012
-sync: db/MANIFEST-000001
-create: wal/000002.log
-sync: wal
-[JOB 1] WAL created 000002
 create: db/temporary.000003.dbtmp
 sync: db/temporary.000003.dbtmp
 close: db/temporary.000003.dbtmp
@@ -83,10 +83,10 @@ sync: wal/000002.log
 close: wal/000002.log
 create: wal/000004.log
 sync: wal
-[JOB 6] WAL created 000004
-[JOB 7] flushing 1 memtable to L0
+[JOB 4] WAL created 000004
+[JOB 5] flushing 1 memtable to L0
 create: db/000005.sst
-[JOB 7] flushing: sstable created 000005
+[JOB 5] flushing: sstable created 000005
 sync: db/000005.sst
 close: db/000005.sst
 sync: db
@@ -96,8 +96,8 @@ sync: db/MANIFEST-000006
 create: db/marker.manifest.000002.MANIFEST-000006
 close: db/marker.manifest.000002.MANIFEST-000006
 sync: db
-[JOB 7] MANIFEST created 000006
-[JOB 7] flushed 1 memtable to L0 [000005] (770 B), in 1.0s (2.0s total), output rate 770 B/s
+[JOB 5] MANIFEST created 000006
+[JOB 5] flushed 1 memtable to L0 [000005] (770 B), in 1.0s (2.0s total), output rate 770 B/s
 
 compact
 ----
@@ -106,10 +106,10 @@ sync: wal/000004.log
 close: wal/000004.log
 reuseForWrite: wal/000002.log -> wal/000007.log
 sync: wal
-[JOB 8] WAL created 000007 (recycled 000002)
-[JOB 9] flushing 1 memtable to L0
+[JOB 6] WAL created 000007 (recycled 000002)
+[JOB 7] flushing 1 memtable to L0
 create: db/000008.sst
-[JOB 9] flushing: sstable created 000008
+[JOB 7] flushing: sstable created 000008
 sync: db/000008.sst
 close: db/000008.sst
 sync: db
@@ -119,12 +119,12 @@ sync: db/MANIFEST-000009
 create: db/marker.manifest.000003.MANIFEST-000009
 close: db/marker.manifest.000003.MANIFEST-000009
 sync: db
-[JOB 9] MANIFEST created 000009
-[JOB 9] flushed 1 memtable to L0 [000008] (770 B), in 1.0s (2.0s total), output rate 770 B/s
-[JOB 9] MANIFEST deleted 000001
-[JOB 10] compacting(default) L0 [000005 000008] (1.5 K) + L6 [] (0 B)
+[JOB 7] MANIFEST created 000009
+[JOB 7] flushed 1 memtable to L0 [000008] (770 B), in 1.0s (2.0s total), output rate 770 B/s
+[JOB 7] MANIFEST deleted 000001
+[JOB 8] compacting(default) L0 [000005 000008] (1.5 K) + L6 [] (0 B)
 create: db/000010.sst
-[JOB 10] compacting: sstable created 000010
+[JOB 8] compacting: sstable created 000010
 sync: db/000010.sst
 close: db/000010.sst
 sync: db
@@ -134,11 +134,11 @@ sync: db/MANIFEST-000011
 create: db/marker.manifest.000004.MANIFEST-000011
 close: db/marker.manifest.000004.MANIFEST-000011
 sync: db
-[JOB 10] MANIFEST created 000011
-[JOB 10] compacted(default) L0 [000005 000008] (1.5 K) + L6 [] (0 B) -> L6 [000010] (770 B), in 1.0s (2.0s total), output rate 770 B/s
-[JOB 10] sstable deleted 000005
-[JOB 10] sstable deleted 000008
-[JOB 10] MANIFEST deleted 000006
+[JOB 8] MANIFEST created 000011
+[JOB 8] compacted(default) L0 [000005 000008] (1.5 K) + L6 [] (0 B) -> L6 [000010] (770 B), in 1.0s (2.0s total), output rate 770 B/s
+[JOB 8] sstable deleted 000005
+[JOB 8] sstable deleted 000008
+[JOB 8] MANIFEST deleted 000006
 
 disable-file-deletions
 ----
@@ -150,10 +150,10 @@ sync: wal/000007.log
 close: wal/000007.log
 reuseForWrite: wal/000004.log -> wal/000012.log
 sync: wal
-[JOB 11] WAL created 000012 (recycled 000004)
-[JOB 12] flushing 1 memtable to L0
+[JOB 9] WAL created 000012 (recycled 000004)
+[JOB 10] flushing 1 memtable to L0
 create: db/000013.sst
-[JOB 12] flushing: sstable created 000013
+[JOB 10] flushing: sstable created 000013
 sync: db/000013.sst
 close: db/000013.sst
 sync: db
@@ -163,17 +163,17 @@ sync: db/MANIFEST-000014
 create: db/marker.manifest.000005.MANIFEST-000014
 close: db/marker.manifest.000005.MANIFEST-000014
 sync: db
-[JOB 12] MANIFEST created 000014
-[JOB 12] flushed 1 memtable to L0 [000013] (770 B), in 1.0s (2.0s total), output rate 770 B/s
+[JOB 10] MANIFEST created 000014
+[JOB 10] flushed 1 memtable to L0 [000013] (770 B), in 1.0s (2.0s total), output rate 770 B/s
 
 enable-file-deletions
 ----
-[JOB 13] MANIFEST deleted 000009
+[JOB 11] MANIFEST deleted 000009
 
 ingest
 ----
 link: ext/0 -> db/000015.sst
-[JOB 14] ingesting: sstable created 000015
+[JOB 12] ingesting: sstable created 000015
 sync: db
 create: db/MANIFEST-000016
 close: db/MANIFEST-000014
@@ -181,9 +181,9 @@ sync: db/MANIFEST-000016
 create: db/marker.manifest.000006.MANIFEST-000016
 close: db/marker.manifest.000006.MANIFEST-000016
 sync: db
-[JOB 14] MANIFEST created 000016
-[JOB 14] MANIFEST deleted 000011
-[JOB 14] ingested L0:000015 (826 B)
+[JOB 12] MANIFEST created 000016
+[JOB 12] MANIFEST deleted 000011
+[JOB 12] ingested L0:000015 (826 B)
 
 metrics
 ----

